src/components/Player.vue

```diff
@@ -124,29 +132,35 @@
             <li @click="changeName">
               <font-awesome-icon icon="user-edit" />Rename
             </li>
-            <li v-if="!session.nomination" @click="nominatePlayer()">
-              <font-awesome-icon icon="hand-point-right" />
-              Nomination
-            </li>
-            <li @click="movePlayer()">
-              <font-awesome-icon icon="redo-alt" />
-              Move player
-            </li>
-            <li @click="swapPlayer()">
-              <font-awesome-icon icon="exchange-alt" />
-              Swap seats
+            <li v-if="!session.nomination" @click="removePlayer">
+              <font-awesome-icon icon="times-circle" />
+              Remove
             </li>
+            <template v-if="!session.nomination">
+              <li @click="movePlayer()">
+                <font-awesome-icon icon="redo-alt" />
+                Move player
+              </li>
+              <li @click="swapPlayer()">
+                <font-awesome-icon icon="exchange-alt" />
+                Swap seats
+              </li>
+              <li @click="toggleOnBlock()">
+                <font-awesome-icon icon="skull" />
+                {{ player.isOnBlock ? "Take off block" : "Put on block" }}
```

Comment:
Seems needed for if+when the ST misclicks? Or else they have to run a whole vote again before they can alter the block.

Reply:
If it really bugs the ST, they could just briefly mark the player as dead or switch to night. Sure, it's still going to happen, but pretty rarely I guess? I wouldn't try to account for *every* possible mistake the ST could make, otherwise the UI would end up super bloated.

Reply:
Fair enough